src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This parser class takes in Strings as input and
+ * convert into an output type based on the requirements
+ *
+ * @author Alex
+ * @version %I%
+ */
+public class Parser {
+
+    /**
+     * Empty constructor for instantiating
+     */
+    public Parser() {
+    }
+
+    /**
+     * This method parse strings to Tasks.
+     *
+     * @param str The string that is being parsed to Tasks, it must have a specific format
+     * @return returns a tas object that is the result of the parsed string
+     */
+    Task parseTxtToTask(String str) {
+        try {
+            String type = str.substring(1, 2);
+            String status = str.substring(4, 5);
+            String descriptionAndDate = str.substring(6);
+            String[] arr = descriptionAndDate.split("\\(");
```

Comment:
Perhaps a name that has a better meaning is more suitable for arrays :)

(It has to be plural too)